shap/plots/_violin.py

```diff
@@ -418,53 +284,23 @@ def violin(shap_values, features=None, feature_names=None, max_display=None, plo
                 y = ys[i, :] / scale
                 c = pl.get_cmap(color)(i / (
                         nbins - 1)) if color in pl.cm.datad else color  # if color is a cmap, use it, otherwise use a color
-                pl.fill_between(x_points, pos - y, pos + y, facecolor=c)
+                pl.fill_between(x_points, pos - y, pos + y, facecolor=c, edgecolor="face")
         pl.xlim(shap_min, shap_max)
 
-    elif not multi_class and plot_type == "bar":
-        feature_inds = feature_order[:max_display]
-        y_pos = np.arange(len(feature_inds))
-        global_shap_values = np.abs(shap_values).mean(0)
-        pl.barh(y_pos, global_shap_values[feature_inds], 0.7, align='center', color=color)
-        pl.yticks(y_pos, fontsize=13)
-        pl.gca().set_yticklabels([feature_names[i] for i in feature_inds])
-
-    elif multi_class and plot_type == "bar":
-        if class_names is None:
-            class_names = ["Class "+str(i) for i in range(len(shap_values))]
-        feature_inds = feature_order[:max_display]
-        y_pos = np.arange(len(feature_inds))
-        left_pos = np.zeros(len(feature_inds))
-
-        if class_inds is None:
-            class_inds = np.argsort([-np.abs(shap_values[i]).mean() for i in range(len(shap_values))])
-        elif class_inds == "original":
-            class_inds = range(len(shap_values))
-        for i, ind in enumerate(class_inds):
-            global_shap_values = np.abs(shap_values[ind]).mean(0)
-            pl.barh(
-                y_pos, global_shap_values[feature_inds], 0.7, left=left_pos, align='center',
-                color=color(i), label=class_names[ind]
-            )
-            left_pos += global_shap_values[feature_inds]
-        pl.yticks(y_pos, fontsize=13)
-        pl.gca().set_yticklabels([feature_names[i] for i in feature_inds])
-        pl.legend(frameon=False, fontsize=12)
-
     # draw the color bar
     if color_bar and features is not None and plot_type != "bar" and \
             (plot_type != "layered_violin" or color in pl.cm.datad):
         import matplotlib.cm as cm
         m = cm.ScalarMappable(cmap=cmap if plot_type != "layered_violin" else pl.get_cmap(color))
         m.set_array([0, 1])
-        cb = pl.colorbar(m, ticks=[0, 1], aspect=1000)
+        cb = pl.colorbar(m, ticks=[0, 1], aspect=80)
         cb.set_ticklabels([labels['FEATURE_VALUE_LOW'], labels['FEATURE_VALUE_HIGH']])
         cb.set_label(color_bar_label, size=12, labelpad=0)
         cb.ax.tick_params(labelsize=11, length=0)
         cb.set_alpha(1)
         cb.outline.set_visible(False)
-        bbox = cb.ax.get_window_extent().transformed(pl.gcf().dpi_scale_trans.inverted())
-        cb.ax.set_aspect((bbox.height - 0.9) * 20)
+        # bbox = cb.ax.get_window_extent().transformed(pl.gcf().dpi_scale_trans.inverted())
+        # cb.ax.set_aspect((bbox.height - 0.9) * 20)
         # cb.draw_all()
```

Comment:
these changes were mostly to align with the `summary_legacy()` function setting. The reason for doing this is so that I can use the same `baseline images` as the `summary_legacy(plot_type-"violin")` tests. We can ensure consistency with the legacy plot this way.

Setting `aspect=80` also led to much more pleasant-looking color bars than `aspect=1000`